content/docs/other-guides/virtual-dev/getting-started-multipass.md

```diff
@@ -4,117 +4,44 @@ description = "Quickly get Kubeflow running locally on native hypervisors"
-$ sudo kubernetes-tools/setup-microk8s.sh
-$ git clone https://github.com/canonical-labs/kubeflow-tools
-$ kubeflow-tools/install-kubeflow.sh
-# Point browser: http://<kubeflow VM IP>:<Ambassador PORT>
-```
-
-Read the next sections for more detail!
-
-## Create a VM with Multipass
-
-[Multipass](https://github.com/CanonicalLtd/multipass) is a general purpose command line interface (CLI) that launches Ubuntu virtual machines based on [cloud-images](http://cloud-images.ubuntu.com/). The benefits of using Multipass include the following:
+If you already have Ubuntu or Linux that supports [snaps](https://snapcraft.io/), you can easily install Kubernetes using [Microk8s](https://microk8s.io/). You can jump to the **Install Kubeflow using Microk8s** section below.
 
-  * quickly create disposable machine learning appliances
-  * leverage the same cloud images locally, reducing surprises when changing from development to staging to production environments in a multi-cloud strategy
-
-Here's a summary of the steps involved:
-
-1. Install the CLI
-2. Use the CLI to create a VM
-3. Enter the VM
-
-### 1. Install Multipass
-
-#### Mac OS X
-
-Install Multipass using the native Mac OS installer:
-
-* Download the latest \*-Darwin.pkg [here](https://github.com/CanonicalLtd/multipass/releases)
-* For more information and documentation, visit the project on [GitHub](https://github.com/CanonicalLtd/multipass)
-
-#### Linux
-
-Install Multipass using [snapcraft](https://snapcraft.io):
-
-```
-$ sudo snap install multipass --beta --classic
-```
-
-### 2. Create an Ubuntu virtual machine
-
-The following command creates a VM with 8GB of memory, 40GB of disk space, and 4 CPU. These are the minimum recommended settings. You are free to adjust them **higher** based on your host machine capabilities and workload requirements.
-```
-$ multipass launch bionic -n kubeflow -m 8G -d 40G -c 4
-```
-
-Note: If you need information on **resource utilization** in the VM, such as memory or disk space or CPU load, you can run ```multipass info kubeflow```
-
-### 3. Enter the VM
-
-```
-$ multipass shell kubeflow
-```
+If you don't have a Linux system already, or you would like to confine your Kubeflow to a disposable machine, then [Create a VM with Multipass](https://multipass.run/) first and then follow the instructions below. That will get you an Ubuntu machine that can be used to install Kubernetes and Kubeflow.
 
 ## Install Kubeflow using Microk8s
 
 Here's a summary of the steps involved:
 
 1. Set up Microk8s
-2. Set up Kubeflow
+2. Enable Kubeflow
 
 ### 1. Install and set up Microk8s
 
-This will install Microk8s if it doesn't already exist, and enable services that are useful for Kubeflow. Please inspect `setup-microk8s.sh` for more information.
-
 ```
-$ git clone https://github.com/canonical-labs/kubernetes-tools
-$ sudo kubernetes-tools/setup-microk8s.sh
+$ snap install microk8s --classic --channel=1.17/stable
```

Comment:
@sarahmaddox It might best to specify the channel as 1.15 as this is listed on https://www.kubeflow.org/docs/started/k8s/overview/ as being the latest currently supported version. I didn't realise a bigger refresh of the instructions was underway, before opening a PR addressing the same thing here: https://github.com/kubeflow/website/pull/1519

Reply:
Thanks @ofaz! 

@ammarn911 Please do incorporate this change into your PR as well.

Reply:
The version (channel) should be 1.14 not 1.15 as in the first comment in this thread. I've approved PR https://github.com/kubeflow/website/pull/1519 which updates the doc to specify channel 1.14.

Reply:
It looks to me as though this PR will overwrite the previous PR which set the channel to 1.14. Please would you set the channel specifically in this PR too, just to ensure it doesn't overwrite the previous one. Apart from that, this LGTM. Thanks for persevering with this change!

Reply:
Hey Sarah, thank you for the review! 

We have tested Kubeflow to be working with MicroK8s 1.17 which is the latest version. Since 1.14 is not supported anymore I'm trying to leave that out. I have left out specifying the version so the command will fetch the latest version of MicroK8s.

Reply:
Thanks for the explanation @ammarn911. In support of your comment, I've looked at the [Microk8s release notes](https://microk8s.io/docs/release-notes) and the notes for MicroK8s 1.17 specifically mention Kubeflow compatibility as a feature.

I'm assuming that MicroK8s 1.17 is based on K8s 1.17.

@jlewi I'm asking for your input here please, as we have two conflicting PRs. The [Kubeflow docs](https://www.kubeflow.org/docs/started/k8s/overview/) don't mention K8s 1.17, but do specifically say Kubeflow does **not** support K8s 1.15 or 1.16. The same page also recommends K8s 1.14. PR https://github.com/kubeflow/website/pull/1519 therefore specifically sets the MicroK8s channel to 1.14.

So, two questions:

1. Should we allow MicroK8s to select the latest version, which is currently 1.17?
1. Should we update the K8s version support in the [Kubeflow docs](https://www.kubeflow.org/docs/started/k8s/overview/)?

Reply:
@ammarn911 could you explain how you tested MicroK8s with K8s 1.17? Last I checked there were still problems with our kustomize manifests kubeflow/manifests#375 

Could you provide some verification that it is working? e.g. can your provide the output of 

```
kubectl -n kubeflow get applications
kubectl -n kubeflow get deploy
```

I'd like to verify that Kubeflow is correctly deploying on 1.17. 

Do we have an issue tracking adding continuous integration for Kubeflow on MicroK8s?

Reply:
@jlewi: MicroK8s is deploying with Juju instead of Kustomize, so not running into the 1.16 issues. The microk8s repo itself has some CI running for `microk8s.enable kubeflow`, if that's what you mean

Reply:
@jlewi, @sarahmaddox please let me know if there's anything else

Reply:
@knkski @ammarn911  It might make sense to discuss this at a community meeting. Agenda is at http://bit.ly/kf-meeting-notes you should be able to comment to add it for an upcoming agenda.

I think my question is do we have a process (and or owners) for ensuring that Kubeflow on multi-pass is tested and "conformant" ? If not; how should we convey this in the docs?

If MicroK8s isn't using the application manifests in kubeflow/manifests; then that suggests all of the manifests in https://github.com/kubeflow/manifests for individual applications need to be ported to Juju. Is that really what you meant? That seems like a fairly large ongoing maintenance cost.  Is that really what's happening and if so what's the plan for keeping them updated?
 
Looking at kubeflow/manifests#375 e.g.
https://github.com/kubeflow/manifests/issues/375#issuecomment-563939595

It looks like many of the issues are that the spec for K8s resources changed in 1.16 and therefore various manifests need to be updated. So kustomize is a big orthogonal. Perhaps you aren't hitting these issues simply because you aren't installing the applications causing problems?

One way to avoid blocking this PR would be to include a banner at the top of the page indicating "Alpha support"  like the one we have on upgrading deployments
https://www.kubeflow.org/docs/upgrading/upgrade/

We can then figure out what is needed in order to feel confident that we can remove the banner.

Reply:
Hey @jlewi, definitely makes sense to discuss in the community meeting, we'll take it there.

In the meantime, I added the Alpha Support banner to roll out the PR and update related docs.